e2e/cypress/integration/scroll/channel_scroll_spec.js

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @scroll
+
+describe('Scroll', () => {
+    let testTeam;
+    let testChannel;
+    let otherUser;
+
+    beforeEach(() => {
+        // # Create new team and new user and visit Town Square channel
+        cy.apiInitSetup().then(({team, channel}) => {
+            testTeam = team;
+            testChannel = channel;
+
+            cy.apiCreateUser().then(({user: user2}) => {
+                otherUser = user2;
+
+                cy.apiAddUserToTeam(testTeam.id, otherUser.id).then(() => {
+                    cy.apiAddUserToChannel(testChannel.id, otherUser.id);
+                });
+            });
+
+            cy.visit(`/${testTeam.name}/channels/${channel.name}`);
+        });
+    });
+
+    it('MM-T2378 Channel with only a few posts opens at the bottom', () => {
+        // # Post a starting message with user 1
+        cy.postMessage('This is the first post');
+
+        // # Post as few as messages so that scroll bar appears as channel name scroll hidden
+        Cypress._.times(8, (postIndex) => {
+            cy.postMessage(`p-${postIndex + 1}`);
+        });
+
+        // # Post the last message
+        cy.postMessage('This is the last post');
+
+        // # Reload the browser
+        cy.reload();
+
+        // * Verify that the top of the channel is scrolled past hidden
+        cy.findByText(`Beginning of ${testChannel.display_name}`).should('exist').and('not.be.visible');
+
+        // * Verify that the last message is visible impling that channel scrolled to bottom on reload
+        cy.findByText('This is the last post').should('exist').and('be.visible');
+    });
+
+    it.only('MM-T2382 Center channel scroll', () => {
+        // # Post a starting message with user 1
+        cy.postMessage('This is the first post');
+
+        // # Make enough posts so that first post is scrolled past hidden
+        Cypress._.times(20, (postIndex) => {
+            cy.postMessage(`p-${postIndex + 1}`);
+        });
```

Comment:
Currently the test failed as shown in this screenshot because there weren't enough posts. Can you please increase the number of posts to 30. It worked fine in that case. 
![Screenshot 2020-09-01 at 5 35 32 PM](https://user-images.githubusercontent.com/1429138/91849627-f727cc80-ec79-11ea-9ebb-90171ff58a06.png)